dev-tools/mage/clean.go

```diff
@@ -12,6 +12,7 @@ import (
 // The values may contain variables and will be expanded at the time of use.
 var DefaultCleanPaths = []string{
 	"build",
+	".agent-testing",
```

Comment:
Is integration:clean a better place to put this?

Reply:
Rooting against it because integration:clean also tears down the ESS stack, which takes a long time to bring back up. It seems more appropriate to clear .agent-testing at the same time as build.

Reply:
This is already removed by integration:clean, but I don't see any issue with doing it in mage:clean as well.